docs/tools/easy_diff.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os, sys
+import subprocess
+from git import cmd
+from tempfile import NamedTemporaryFile
+
+SCRIPT_PATH = os.path.abspath(__file__)
+CLICK_HOUSE_REPO_HOME = os.path.join(os.path.dirname(SCRIPT_PATH), '../../')
+
+
+def diffFile(reference_file, working_file, git, temp_diff):
```

Comment:
```suggestion
def diff_file(reference_file, working_file, git, temp_diff):
```
snake_case style is more common in python for function names and as far as I remember nearby scripts use it, let's change this? (here and below)